src/app/starredentries/starredentries.component.html

```diff
@@ -1,253 +1,269 @@
+                        }}
+                      </h4>
+                    </mat-card-title>
+                  </mat-card-header>
+                </a>
+                <app-starring class="pull-right" [workflow]="workflow" (starGazersChange)="starGazersChange()"></app-starring>
+              </div>
               <mat-card-header>
-                <mat-card-title fxLayoutAlign="flex-start" fxLayoutGap="1rem">
-                  <img src="../../../assets/svg/sub-nav/tool.svg" alt="tool logo" />
-                  <h4>{{ tool?.namespace + '/' + tool?.name + (tool?.toolname ? '/' + tool?.toolname : '') }}</h4>
-                </mat-card-title>
-                <div *ngIf="tool?.starredUsers.length !== 0" fxLayout="row" fxLayoutAlign="end center"></div>
+                <div *ngIf="workflow?.starredUsers.length !== 0" fxLayout="row" fxLayoutAlign="end center"></div>
+                <mat-card-subtitle class="truncate-text-1">{{ workflow?.description }}</mat-card-subtitle>
               </mat-card-header>
-            </a>
-            <app-starring class="pull-right" [tool]="tool" (starGazersChange)="starGazersChange()"></app-starring>
-          </div>
-          <div>
-            <hr />
-            <div fxLayout="row" fxLayoutAlign="space-between stretch">
-              <div fxLayout="row" fxLayoutAlign="flex-start" fxLayoutGap="1rem">
-                <small class="spacing">Last updated {{ tool?.lastUpdated | date }}</small>
-                <mat-chip-list>
-                  <mat-chip class="bubble container-background" *ngFor="let label of tool?.labels | slice: 0:3">{{ label.value }}</mat-chip>
-                  <mat-chip *ngIf="tool?.versionVerified" matTooltip="Verified" class="bubble">
-                    <a class="verified-check" fxLayoutGap="0.5rem" fxLayoutAlign=" center">
-                      <mat-icon class="verified-icon">done</mat-icon>
-                      <span>Verified</span>
-                    </a>
-                  </mat-chip>
-                  <mat-chip class="bubble" *ngFor="let descriptortype of tool?.descriptorType">
-                    {{ descriptortype }}
-                  </mat-chip>
-                </mat-chip-list>
+              <div>
+                <hr />
+                <div fxLayout="row" fxLayoutAlign="space-between stretch">
+                  <div fxLayout="row" fxLayoutAlign="flex-start" fxLayoutGap="1rem">
+                    <small class="spacing">Last updated {{ workflow?.lastUpdated | date }}</small>
+                    <div fxLayout="row wrap" fxLayoutGap="1rem">
+                      <span>
+                        <mat-chip-list>
+                          <mat-chip class="bubble workflow-background" *ngFor="let label of workflow?.labels | slice: 0:3">{{
+                            label.value
+                          }}</mat-chip>
+                          <mat-chip class="bubble" *ngIf="workflow.versionVerified === true">
```

Comment:
Inconsistent use of `?` -- used on lines 79 and 83 but not here and line 95 (probably inherited). If it's needed, it's needed everywhere; if it's not needed anywhere. Not just here, but portions of the code as well.

If this is existing code that just got moved, don't worry about it.

Reply:
Yeah this is just code that got moved!